src/rust/dbv2/migrations/2024-01-11-173201_enumerated_volume/down.sql

```diff
@@ -0,0 +1,11 @@
+-- This file should undo anything in `up.sql`
```

Comment:
I suggest you merge in main then try out the new migrations checker script

Comment here if it works?

Reply:
Run locally on my machine just now